src/components/ModelingSidebar/ModelingPanes/KclEditorMenu.module.css

```
@@ -3,6 +3,7 @@
   @apply font-mono !no-underline text-xs font-bold select-none text-chalkboard-90;
   @apply ui-active:bg-primary/10 ui-active:text-primary ui-active:text-inherit;
   @apply transition-colors ease-out;
+  margin: 0;
```

Is there a better way than this @franknoirot?

---

```suggestion
  @apply m-0;
```
You could add `m-0` among `px-2 py-1` since they're also bounding box-related, but this works fine

---

Ha nice I'll apply this!